docs/frontend/backend-layouts.md

```diff
@@ -39,38 +43,28 @@ export default {
   extends: T3BlDefault
 }
 </script>
-
 ```
 
+
 ## Register backend layouts
 
-You have to register component layouts to Nuxt application context - this is needed to provide the backend layout for SSR purposes. Just use `registerBackendLayouts` plugin
-   
-create: `plugins/layouts.js`:
+You can register backend layout as a regular global Vue component. 
+
+edit/create: `plugins/components.js`:
 
 ```js
 import Vue from 'vue'
-import { registerBackendLayouts } from '~typo3/plugins/layouts'
-import T3Bl2Columns from '~/components/T3Bl2Columns'
-const layouts = {
-  T3Bl2Columns
-}
-
-export default ({ app }) => {
-  Vue.use(registerBackendLayouts, {
-    context: app,
-    layouts
-  })
-}
+import Bl2Columns from '~/components/Bl2Columns'
+Vue.component('T3Bl2Columns', Bl2Columns)
```

Comment:
The naming of the component and file should be consistent. Maybe we should remove `T3`? Let's discuss.

Reply:
I changed my mind, this is the best option. You shouldn't name non-native typo3 components like this.

Reply:
let's stay with prefixes only for registration, T3 is needed for overriding nuxt-typo3 components, the user doesn't have to include prefixes in their own components.